dummy-project/src/modeling/evaluate.py

```
@@ -0,0 +1,11 @@
+from sklearn.metrics import accuracy_score
```

Ruff (0.8.2)

`sklearn.metrics.accuracy_score` imported but unused

Remove unused import: `sklearn.metrics.accuracy_score`

(F401)